src/systemctl/systemctl.c

```diff
@@ -7708,7 +7713,7 @@ static int systemctl_help(void) {
                "  emergency                           Enter system emergency mode\n"
                "  halt                                Shut down and halt the system\n"
                "  poweroff                            Shut down and power-off the system\n"
-               "  reboot [ARG]                        Shut down and reboot the system\n"
+               "  reboot --reboot-arg=[ARG]           Shut down and reboot the system\n"
```

Comment:
options (i.e. those argument starting with `--`) should be documented in the Options section below. Hence remove the whole reboot argument from here